src/components/About/About.js

```diff
@@ -11,6 +13,12 @@ import arrow from './images/arrow.svg'
 import styles from './About.module.scss'
 
 const About = () => {
+  const [isMobile, setIsMobile] = useState(false)
+
+  useEffect(() => {
+    return IsMobile.subscribe(value => setIsMobile(value))
+  }, [])
```

Comment:
Make it as a custom hook, that is stored in `./src/hooks`. This will allow us to reuse in other places, e.g. `src/pages/index.js`